test/e2e/checkpoint_test.go

```diff
@@ -977,6 +977,9 @@ var _ = Describe("Podman checkpoint", func() {
 	})
 
 	It("podman checkpoint container with --pre-checkpoint", func() {
+		if podmanTest.Host.Arch == "arm64" {
+			Skip("skip on arm64/aarch64, checkpoint-restore/criu GH issue #2676")
```

Comment:
could as well put the full URL here instead so users don't have to manually look it up.